lib/invoker/power/power.rb

```diff
@@ -1,3 +1,31 @@
 require "invoker/power/http_response"
 require "invoker/power/dns"
 require "invoker/power/balancer"
+
+require "invoker/power/tld"
+
+module Invoker
+  module Power
+    class << self
+      def set_tld(tld_value)
+        @tld_value = tld_value
+      end
+
+      def reset_tld
+        @tld_value = nil
+      end
+
+      def tld
+        tld_value = @tld_value
+        if !tld_value && Invoker.config
+          tld_value = Invoker.config.tld
+        end
+        Tld.new(tld_value)
+      end
+
+      def tld_value
+        tld.value
+      end
+    end
+  end
```

Comment:
This is really unnecessary. Just have `tld` is a small string. Just store it as a part of `Setup` class ?